iss_forecasting/analysis/utils/time_series.py

```diff
@@ -0,0 +1,76 @@
+"""Analysis utils for generating and manipulating time series data
+"""
+import numpy as np
+import pandas as pd
+
+
+def generate_ts(
+    period: str,
+    lag: int,
+    start: str = "01/01/2007",
+    end: str = "12/31/2021",
+    rf_scale: float = 100,
+    rf_const: float = 200,
+    factor: float = 1.1,
+    pi_scale: float = 2,
+) -> pd.DataFrame:
+    """Generate dataframe containing columns for time_period, research_funding_total,
+    investment_raised_total and tech_category. research_funding_total and
+    investment_raised_total have generated time series data, where investment_raised_total
+    is research_funding_total shifted by the lag variable.
```

Comment:
Could you please add a brief explanation of how the time series are generated?
(I guess you have something like first-order autoregressive process here, but you've also injected random noise into the coefficient and constant)